postcss.config.js

```diff
@@ -1,7 +1,20 @@
+const autoprefixer = require('autoprefixer');
+const assets = require('postcss-assets');
+
+const CF_DIST = process.env.CLOUDFRONT_DIST;
```

Comment:
I think that would be a good idea. URLs in CSS is kind of messy in general IMO.

I think in general we should move sometime soon towards extracting our CSS into separate files, instead of leaning on the Webpack loader, in production (using the extract text plugin or similar). Moving these URLs out of our `scss` code would be a step in that direction.